draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4059,7 +3742,7 @@ role. The content of the attribute is a 64-bit unsigned integer in
 network byte order, which contains a random number. The number is used
 for solving role conflicts, when it is referred to as the tie-breaker
 value. An ICE agent MUST use the same number for all Binding requests,
-for all streams, within an ICE session. The ICE agent MAY change
```

Comment:
This is just fixing it to make it consistent with sec-recv-response, which says " The ICE agent MUST NOT change the tie-breaker value."  I think this was just an oversight when making that change.